### DISCUSSION TITLE
Disable email notifications when a collection is created

### DISCUSSION THREAD

**Discussion by samstride** [outline/outline#7537] [General] [answered]
Hi,

Sometimes there is a need to create a confidential collection that only admins or certain users can see.

On v0.78.0, every time a confidential collection is created with the `No access` permission, users who have `Collection created` notification turned on get an email that a (confidential) collection has been created.

Are there any strategies to suppress collection created notifications?

Or is the solution to have an option during collection creation to disable notifications?

Thanks.

**Comment by tommoor** [comment]
That sounds like a bug, we shouldn't notify anyone of a collection they cannot access.

**Reply by samstride** [reply, marked as answer]
Thanks for the super quick reply.

We have double checked and if the collection is created with permission `no access`, no email is sent.

Sorry, user error. Can be closed.
